README.md

```diff
@@ -101,6 +101,35 @@ Finally, run `nix-build`, and voilà, in `./result/` you find the project with
 all its dependencies correctly linked to their corresponding `node_modules`
 folder, recursively.
 
+## Using private package repository
+
+Since `yarn2nix` uses standard `fetchurl` to download packages,
+it is possible to authenticate by overriding `fetchurl` to use
+`netrc` file.
+
+```nix
+let
+  auth = { login: "your_login"; taken "your_auth_token"; };
+  config = {
+    packageOverrides = pkgs: rec {
+      fetchurl = otps: pkgs.fetchurl (opts // {
+        netrcPhase = ''
+          cat << EOF > /build/netrc
+            machine repository_url
+              login ${auth.login}
+              password ${auth.token}
```

Comment:
This puts the `auth` login/token in the (public) nix store.

Reply:
Thanks I didn't realized that